ivy/functional/frontends/numpy/statistics/averages_and_variances.py

```diff
@@ -2,6 +2,7 @@
 import ivy
 from ivy.func_wrapper import from_zero_dim_arrays_to_float
 from ivy.functional.frontends.numpy.func_wrapper import to_ivy_arrays_and_back
+import pytest
```

Comment:
We don't import pytest into the frontends directly provided your not using pytest.mark_parametrize.We just rely on hypothesis testing.